File: 1768. Merge Strings Alternately/Solution1768.java
class Solution1768 {
    public static String mergeAlternately(String word1, String word2) {
        int min = Math.min(word1.length(), word2.length());
        StringBuilder sb = new StringBuilder();
        for (int i = 0; i < min; i++) {
            sb.append(word1.charAt(i));
            sb.append(word2.charAt(i));
        }
        if (word1.length() > word2.length()) {
            sb.append(word1.substring(min));
        } else {
            sb.append(word2.substring(min));
        }
        return sb.toString();
    }
}

public static void main(String[] args) {
    System.out.println(Solution1768.mergeAlternately("abc", "pqr"));
}

File: 1768. Merge Strings Alternately/Solution1768.py
class Solution(object):
    def mergeAlternately(self, word1, word2):
        
        minimum = min(len(word1),len(word2))
        string = ""
        for x in range(minimum):
            string += word1[x]
            string += word2[x]
   
        if len(word1) > len(word2):
            string += word1[minimum:]
        else:
            string += word2[minimum:]
        return string
    
c = Solution()
print (c.mergeAlternately("abc","pqr"))       